### ISSUE TITLE
Error messages on startup (using Ubuntu 7.04)

### ISSUE THREAD

**Issue by GoogleCodeExporter** [deanlaw/cfcgenerator#30]
```
What steps will reproduce the problem?

Everytime I'm accessing http://localhost/cfcgenerator/index.cfm the
following errors occur:
"flex/messaging/io/amf/UnknownTypeException"
"Server.Processing"
"Fault occured in GetTemplatesCommand."
"flex/messaging/io/amf/UnknownTypeException"
"Server.Processing"
"Fault occured in GetDatasourcesCommand."



What is the expected output? What do you see instead?

After clicking away those annoying popup messages, the flex interface is
basically working fine, but I can't select any datasources, tables and
templates.



What version of the product are you using? On what operating system?

I'm using 1.0 RC1, ColdFusion MX 7.02 and MySQL 5 on Ubuntu Linux (Feisty
7.04).
```

Original issue reported on code.google.com by `alexander.grueneberg` on 12 Apr 2007 at 7:06
